assets/scripts/AlgoliaScripts/.gitignore

```diff
@@ -0,0 +1,116 @@
+# Logs
```

Comment:
Please remove this .gitignore file

Reply:
Any specific reason ? If it's not there we might end up adding unnecessary node_modules.

Reply:
Oh, do folks on the team do `git add .` a lot instead of just adding explicitly specified files they changed? We would not accept any PRs that were adding anything from node_modules directory